numba_dpex/tests/dpjit_tests/dpnp/test_dpnp_empty_like.py

```diff
@@ -16,23 +16,20 @@
 shapes = [10, (2, 5)]
 dtypes = [dpnp.int32, dpnp.int64, dpnp.float32, dpnp.float64]
 usm_types = ["device", "shared", "host"]
-devices = ["cpu", "unknown"]
 
 
 @pytest.mark.parametrize("shape", shapes)
 @pytest.mark.parametrize("dtype", dtypes)
 @pytest.mark.parametrize("usm_type", usm_types)
-@pytest.mark.parametrize("device", devices)
-def test_dpnp_empty_like(shape, dtype, usm_type, device):
+def test_dpnp_empty_like_from_device(shape, dtype, usm_type):
+    device = dpctl.SyclDevice().filter_string
+
     @dpjit
     def func(a):
         c = dpnp.empty_like(a, dtype=dtype, usm_type=usm_type, device=device)
         return c
 
-    if isinstance(shape, int):
-        NZ = numpy.random.rand(shape)
-    else:
-        NZ = numpy.random.rand(*shape)
+    NZ = dpnp.empty(shape)
```

Comment:
The idea was to test if we can create an "empty" array with the same shape from a "non-empty" array. Now `dpnp.random.random()` works, we can use that.

Reply:
I do not think it matters. Let us use `dpnp.ones` if you want a non-empty array. Does `dpnp.random.random` work on all architectures?